galaxy_ng/tests/integration/conftest.py

```diff
@@ -97,6 +97,11 @@ class AnsibleConfigFixture(dict):
             "password": "professor",
             "token": "abcdefghijklmnopqrstuvwxyz1234567894",
         },
+        "ee_admin": {
+            "username": "ee-admin",
```

Comment:
you can probably make these both `ee_admin` ... the only reason some usernames have a dash is due to their setup in keycloak